src/main/java/andrew/BuildingGame/Game/Game.java

```diff
@@ -121,6 +122,10 @@ public void advanceGamePhase() {
     if (gameState == GameStateManager.GameState.BUILD) { Util.sendMessageToPlayers(participants, ChatColor.GOLD + "Time to build!"); }
     if (gameState == GameStateManager.GameState.GUESS) { Util.sendCustomJsonMessage(participants, JsonStrings.generateGuessText()); }
 
+    if (isFinalBuildingRound) {
+      Util.sendCustomJsonMessage(participants, JsonStrings.generateFinalBuildingRoundText());
+    }
```

Comment:
Minor: I would squash this to one line since the conditional body is one line

i.e. `if (isFInalBuildingRound) { Util.sendCustomJsonMessage(participants, JsonStrings.generateFinalBuildingRoundText()); }`

This way it also matches the conditional checks above